src/xlsx/functions/charts.ts

```diff
@@ -204,6 +204,13 @@ function addBarChart(chart: ExcelChartDefinition): XMLString {
   //
   // overlap and gapWitdh seems to be by default at -20 and 20 in chart.js.
   // See https://www.chartjs.org/docs/latest/charts/bar.html and https://www.chartjs.org/docs/latest/charts/bar.html#barpercentage-vs-categorypercentage
+
+  // Excel does not support aggregating labels
+  // GSheets does not export the chart if it aggregates the labels
+  if (chart.aggregated) {
```

Comment:
how would you end up with a chart where aggregated is `true` ?
Aren't they filtered in `exportForExcel`?